### ISSUE TITLE
Python binary wheel doesn't get picked under Linux/ARM64

### ISSUE THREAD

**Issue by sacundim** [hex-inc/vegafusion#356]
To reproduce, try building this Dockerfile for the `linux/arm64` platform (e.g. on an Apple Silicon Mac, or in QEMU):

```Dockerfile
FROM python:3.10-slim
RUN pip install "vegafusion[embed]==1.3.0"
```

The RUN step fails with the error below.

Notably, the `pip install` works fine on native MacOS ARM64.

```
 => ERROR [linux/arm64 2/2] RUN pip install "vegafusion[embed]==1.3.0"                                                                                          163.7s
------                                                                                                                                                                 
 > [linux/arm64 2/2] RUN pip install "vegafusion[embed]==1.3.0":                                                                                                       
#0 36.75 Collecting vegafusion[embed]==1.3.0
#0 39.31   Downloading vegafusion-1.3.0-py3-none-any.whl (23 kB)
#0 46.68 Collecting pandas
#0 46.74   Downloading pandas-2.0.3-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (11.6 MB)
#0 49.65      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.6/11.6 MB 4.6 MB/s eta 0:00:00
#0 51.49 Collecting altair>=4.2.0
#0 51.73   Downloading altair-5.0.1-py3-none-any.whl (471 kB)
#0 51.87      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 471.5/471.5 kB 4.8 MB/s eta 0:00:00
#0 55.31 Collecting psutil
#0 55.40   Downloading psutil-5.9.5.tar.gz (493 kB)
#0 55.54      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 493.5/493.5 kB 5.1 MB/s eta 0:00:00
#0 56.73   Installing build dependencies: started
#0 96.98   Installing build dependencies: finished with status 'done'
#0 97.00   Getting requirements to build wheel: started
#0 104.5   Getting requirements to build wheel: finished with status 'done'
#0 104.5   Preparing metadata (pyproject.toml): started
#0 109.7   Preparing metadata (pyproject.toml): finished with status 'done'
#0 113.2 Collecting pyarrow>=5
#0 113.5   Downloading pyarrow-12.0.1-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (36.4 MB)
#0 122.4      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 36.4/36.4 MB 4.0 MB/s eta 0:00:00
#0 126.4 Collecting vegafusion-python-embed==1.3.0
#0 126.5   Downloading vegafusion_python_embed-1.3.0.tar.gz (6.5 MB)
#0 127.7      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.5/6.5 MB 5.7 MB/s eta 0:00:00
#0 138.2   Installing build dependencies: started
#0 159.6   Installing build dependencies: finished with status 'done'
#0 159.6   Getting requirements to build wheel: started
#0 160.8   Getting requirements to build wheel: finished with status 'done'
#0 160.8   Preparing metadata (pyproject.toml): started
#0 162.1   Preparing metadata (pyproject.toml): finished with status 'error'
#0 162.2   error: subprocess-exited-with-error
#0 162.2   
#0 162.2   × Preparing metadata (pyproject.toml) did not run successfully.
#0 162.2   │ exit code: 1
#0 162.2   ╰─> [6 lines of output]
#0 162.2       
#0 162.2       Cargo, the Rust package manager, is not installed or is not on PATH.
#0 162.2       This package requires Rust and Cargo to compile extensions. Install it through
#0 162.2       the system's package manager or via https://rustup.rs/
#0 162.2       
#0 162.2       Checking for Rust toolchain....
#0 162.2       [end of output]
#0 162.2   
#0 162.2   note: This error originates from a subprocess, and is likely not a problem with pip.
#0 162.3 error: metadata-generation-failed
#0 162.3 
#0 162.3 × Encountered error while generating package metadata.
#0 162.3 ╰─> See above for output.
#0 162.3 
#0 162.3 note: This is an issue with the package mentioned above, not pip.
#0 162.3 hint: See above for details.
```

**Comment by jonmmease** [comment]
Thanks for the report @sacundim! Yeah, we need to build wheels for linux arm64. I'll try to get that working for the coming 1.4.0 release.

**Comment by jonmmease** [comment]
I think I got this working in #358.

Could you try installing this wheel and see if it works for your case?
[vegafusion_python_embed-1.4.0rc2-cp38-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl](https://filedn.com/lo5VE4SmtWKXIvNsinHVy7F/packages/vegafusion_python_embed-1.4.0rc2-cp38-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl)

**Comment by sacundim** [comment]
This Dockerfile works fine:

```
FROM python:3.10-slim
RUN pip install https://filedn.com/lo5VE4SmtWKXIvNsinHVy7F/packages/vegafusion_python_embed-1.4.0rc2-cp38-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
```

...but I haven't figured out how to install this as the "embed" extra to in the `vegafusion[embed]==1.4.0rc2` dependency spec in a Poetry pyproject.toml. I figure that just the Dockerfile working is enough confirmation here, though

**Comment by jonmmease** [comment]
> I figure that just the Dockerfile working is enough confirmation here, though

Yeah, I think that's enought. Thanks for checking!

**Comment by jonmmease** [comment]
linux-arm64 wheel published for 1.4.0-rc3

**Comment by sacundim** [comment]
> linux-arm64 wheel published for 1.4.0-rc3

Nice, thanks, I'll give it a try in a few days
